experimental/tiledb/common/dag/nodes/detail/simple/function.h

```diff
@@ -103,14 +105,18 @@ class FunctionNode : public GraphNode,
    * @tparam The type of the function (or function object) that accepts items.
    */
   template <class Function>
-  explicit FunctionNode(
-      Function&& f,
+  explicit FunctionNode(Function&& f
+#if 0 
```

Comment:
Why are we commenting this out rather than just deleting the code?

Reply:
As a placeholder to come back to.  This might not be good/company practice.  Should have put a `@todo` on it.  This also should perhaps be `#ifndef _MSCVER` rather than #if 0.  I will take another quick look at this.

Reply:
The other aspect of the `@todo` is to just use concepts instead of `enable_if`....